cmd/nerdctl/container_run_linux_test.go

```diff
@@ -503,3 +503,22 @@ func TestRunWithDetachKeys(t *testing.T) {
 	container := base.InspectContainer(containerName)
 	assert.Equal(base.T, container.State.Running, true)
 }
+
+func TestRunDeviceCDI(t *testing.T) {
+	cwd, err := os.Getwd()
+	assert.NilError(t, err)
+	cdiSpecDir := filepath.Join(cwd, "testdata", "cdi")
+
+	os.Setenv("CDI_SPEC_DIRS", cdiSpecDir)
```

Comment:
I have added a basic test that sets the `CDI_SPEC_DIRS` option via an envvar. Do you want me to add one that sets it via the command line argument too?

Reply:
Yes, that would be nice.
It should be also supported via TOML: https://github.com/containerd/nerdctl/blob/main/docs/config.md

Reply:
https://github.com/containerd/nerdctl/blob/370cab6ef2b8f7abe61d3661629ab8d74ecf2f4c/cmd/nerdctl/main_test.go#L46-L73